$NetBSD: patch-aa,v 1.4 2015/05/31 15:25:41 taca Exp $

* Add support for DESTDIR.
* Fix build problem on Ruby 2.2 and later.

--- install.rb.orig	2008-10-22 15:53:45.000000000 +0000
+++ install.rb
@@ -1,10 +1,15 @@
 require 'rbconfig'
+require 'fileutils'
 
-sitelibdir = Config::CONFIG.fetch("sitelibdir")
-bindir = Config::CONFIG.fetch("bindir")
+sitelibdir = RbConfig::CONFIG.fetch("sitelibdir")
+bindir = RbConfig::CONFIG.fetch("bindir")
 
 def copy(src, dest, mode)
+  if (destdir = ENV['DESTDIR'])
+    dest = File.join(destdir, dest)
+  end
   p dest
+  FileUtils::mkdir_p(File.dirname(dest))
   open(src, "rb") do |infile|
     open(dest, "wb") do |outfile|
       while buf = infile.read(8192)
